packages/core/src/__tests__/uuid.test.ts

```diff
@@ -1,24 +1,11 @@
-import { NativeModules } from 'react-native';
 import { getUUID } from '../uuid';
+import uuid from 'react-native-uuid';
 
 describe('#uuid', () => {
-  beforeEach(() => {
-    NativeModules.AnalyticsReactNative = {
-      getUUIDSync: () => {},
-    };
-  });
-
-  it('should get a UUID from the bridge in prod', () => {
-    const oldDevValue = __DEV__;
-    (global as any).__DEV__ = false;
-    const id = 'nativeUUID-123';
-    jest
-      .spyOn(NativeModules.AnalyticsReactNative, 'getUUIDSync')
-      .mockReturnValueOnce(id);
-
-    const result = getUUID();
+  it('should get a UUID from react-native-uuid', () => {
+    const uuidSpy = jest.spyOn(uuid, 'v4');
 
-    expect(result).toBe(id);
-    (global as any).__DEV__ = oldDevValue;
+    getUUID();
+    expect(uuidSpy).toBeCalled();
   });
```

Comment:
I think we can remove this test. Not sure what it was trying to accomplish besides making sure we were calling a NativeModule method.

Reply:
awesome thank you! I kinda thought so too haha. I'll remove it right now then.